auctions/views.py

```diff
@@ -116,7 +116,7 @@ def post(self, request, *args, **kwargs):
                 if claim.payout_request():
                     messages.success(request, 'Your payout was sent.')
                 else:
-                    messages.error(request, "Your payout has been requested.")
+                    messages.error(request, "Sorry please try later")
```

Comment:
Maybe instead of `payout_request()` returning `True|False` we have it return the `Claim`/`self` object, and use api_success or another object field to decide what to show the user.